### DISCUSSION TITLE
Can't create a sub-folder in the Macintosh version.

### DISCUSSION THREAD

**Discussion by rhedin** [phcode-dev/phoenix#1993] [General]
Hi.   In the web version of the editor, I can right-click (control-click) a folder, and create a subfolder.   In the version on my laptop, the new folder is always at the same level as the original folder.  I am using an Apple M3 Pro running Sonoma 14.6.1. 

Am I just doing something stupid, or is the Mac version not yet reliable, while the online version is?

**Comment by abose** [comment]
Thanks for reporting the issue @rhedin 

This is fixed via https://github.com/phcode-dev/phoenix/pull/1994 today and will be available in the next release this month end or early next month. 

This is a mac desktop and safari only issue with `ctrl-click` based context menu activation. Context menu activation with right mouse button click will work as expected. Looks like all our team members have configured to use a dedicated right mouse button and this missed our QA passes. I am a bit surprised that no one reported this issue even after 9 months and over 100,000 desktop users in mac having used it at this point. Many thanks for reporting this issue!

Please see https://github.com/phcode-dev/phoenix/pull/1994 to find out why it happened.

**Comment by rhedin** [comment]
Thanks for the clear explanation, @abose.  Both here and in the merge request.   And thanks for the fix.
